hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/common/statemachine/commandhandler/TestBlockDeletion.java

```diff
@@ -817,6 +829,7 @@ public void testBlockDeleteCommandParallelProcess() throws Exception {
     // Wait for block delete command sent from OM
     GenericTestUtils.waitFor(() -> {
```

Comment:
can this be replaced by waitForDeletedBlockLog()?
Also it loosk like the next GenericTestUtils.waitFor() can be replaced by waitBlockDeleted()